test/test_prototype_datasets_api.py

```diff
@@ -208,7 +193,7 @@ def test_default_config(self):
         ("config", "kwarg"),
         [
             pytest.param(*(datasets.utils.DatasetConfig(split="test"),) * 2, id="specific"),
-            pytest.param(make_minimal_dataset_info().default_config, None, id="default"),
+            pytest.param(datasets.utils.DatasetConfig(split="train"), None, id="default"),
```

Comment:
just wondering if the `id="default"` is still accurate?

Reply:
I wondered the same thing re-reading. Let me check.

Reply:
The test implicitly relied on the setup of the mock dataset. This is fixed now.